dis_snek/api/http/http_requests/guild.py

```diff
@@ -206,7 +212,14 @@ async def get_guild_bans(self, guild_id: "Snowflake_Type") -> List[discord_typin
             List of ban objects
 
         """
-        return await self.request(Route("GET", f"/guilds/{guild_id}/bans"))
+        params = dict_filter_none(
```

Comment:
Should filter missing instead

Reply:
It looks like missing was already being filtered.

Reply:
ohyea, its handle by http.request thing, then there isn't a need to filter none

Reply:
There is a need to `filter_none`. the libary filters out `MISSING` not `None` as `None` could be an expected payload.

Reply:
Re-opening this. Soz. After reading the new docs, this filter isnt required at all.